crates/graphql/src/schema.rs

```diff
@@ -555,6 +556,130 @@ impl<'a> From<models::Storefront<'a>> for Storefront {
     }
 }
 
+#[derive(Debug, Clone)]
+struct Marketplace {
+    subdomain: String,
+    name: String,
+    description: String,
+    logo_url: String,
+    banner_url: String,
+    auction_house_address: String,
+}
+
+impl<'a> From<models::StoreConfigJson<'a>> for Marketplace {
+    fn from(
+        models::StoreConfigJson {
+            config_address: _,
+            name,
+            description,
+            logo_url,
+            banner_url,
+            subdomain,
+            owner_address: _,
+            auction_house_address,
+            ..
+        }: models::StoreConfigJson,
+    ) -> Self {
+        Self {
+            subdomain: subdomain.into_owned(),
+            name: name.into_owned(),
+            description: description.into_owned(),
+            logo_url: logo_url.into_owned(),
+            banner_url: banner_url.into_owned(),
+            auction_house_address: auction_house_address.into_owned(),
+        }
+    }
+}
+
+#[juniper::graphql_object(Context = AppContext)]
+impl Marketplace {
+    pub fn subdomain(&self) -> String {
+        self.subdomain.clone()
+    }
+
+    pub fn name(&self) -> String {
+        self.name.clone()
+    }
+
+    pub fn description(&self) -> String {
+        self.description.clone()
+    }
+
+    pub fn logo_url(&self) -> String {
+        self.logo_url.clone()
+    }
+
+    pub fn banner_url(&self) -> String {
+        self.banner_url.clone()
+    }
+
+    pub fn auction_house_address(&self) -> String {
+        self.auction_house_address.clone()
+    }
+
+    pub async fn auction_house(&self, context: &AppContext) -> Vec<AuctionHouse> {
+        context
+            .auction_house_loader
+            .load(self.auction_house_address.clone())
+            .await
+    }
+}
+
+#[derive(Debug, Clone)]
+struct AuctionHouse {
+    address: String,
+    authority: String,
+    seller_fee_basis_points: i32,
+    auction_house_fee_account: String,
+}
+
+#[juniper::graphql_object(Context = AppContext)]
+impl AuctionHouse {
+    pub fn address(&self) -> String {
+        self.address.clone()
+    }
+
+    pub fn authority(&self) -> String {
+        self.authority.clone()
+    }
+
+    pub fn seller_fee_basis_points(&self) -> i32 {
+        self.seller_fee_basis_points.into()
```

Comment:
```suggestion
        self.seller_fee_basis_points
```

Reply:
I added this because the seller_fee_basis_points was i16 in db and I had to create a custom scalar for that but it is working without into() xD

Reply:
idk why I added into() here :D

Reply:
All good, i'm just relaying the clippy warnings from CI lol